src/style.css

```diff
@@ -1 +1,107 @@
 /* Styles go here */
+html {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+body {
+  margin: 0;
+  min-width: 1024px;
+}
+
+.header {
+  position: sticky;
+  top: 0;
+  --height: 60px;
```

Comment:
I would suggest the next tip:
- write `:root {...}` at the top of your stylesheet
- move all variables there
- use them with `var(...)`